pkg/detectors/satismeterwritekey/satismeterwritekey.go

```diff
@@ -20,7 +25,9 @@ var (
 	client = common.SaneHttpClient()
 
 	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
-	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"satismeter"}) + `\b([a-z0-9A-Z]{16})\b`)
+	writeKeyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"satismeter"}) + `\b([a-z0-9A-Z]{32})\b`)
```

Comment:
The writeKey I received after creating my account is 32 characters long. Can anyone else confirm this to help validate it?

Reply:
confirmed!